lisp/init-scala-metals.el

```diff
@@ -0,0 +1,43 @@
+;; init-web.el --- Initialize web configurations.	-*- lexical-binding: t -*-
```

Comment:
The docs should be updated.

Reply:
Ok. I will push a new version for merge.